Documentation/gettingstarted/host-firewall.rst

```diff
@@ -22,24 +22,45 @@ Enable the Host Firewall in Cilium
 
 Deploy Cilium release via Helm:
 
-.. parsed-literal ::
+.. code:: shell-session
 
-    helm install cilium |CHART_RELEASE|        \\
+    $ helm install cilium |CHART_RELEASE|      \\
```

Comment:
Unless there's a mix of commands and command outputs, I would stick to the current formatting. Adding the shell sign messes up copy pasta and the Copy all/Copy first line button of read the docs; I don't think it's worth the additional consistency.

Side note: Might be nice to have a policy on how we handle that across the docs.

Reply:
Ok. And I think I should actually keep this block as a `parsed-literal`, or `|CHART_RELEASE|` may not be interpreted? I need to double check.

Reply:
Heads-up: this block _must_ remain as a `parsed-literal` for the `|CHART_RELEASE|` to be parsed.